lib/data/tools/pen.dart

```diff
@@ -72,6 +72,13 @@ class Pen extends Tool {
     _currentPen = currentPen;
   }
 
+  bool isPressureEnabled() {
+    return pressureEnabled;
+  }
+  double getSize() {
+    return options.size;
+  }
+
```

Comment:
These seem like redundant getters, we can just use the underlying fields themselves